build-tools/packages/build-cli/src/commands/report/codeCoverage.ts

```diff
@@ -158,8 +175,18 @@ export default class ReportCodeCoverageCommand extends BaseCommand<
 		);
 
 		// Fail the build if the code coverage analysis shows that a regression has been found.
-		if (!success) {
+		if (!(success || skipBuildFailureOnRegression)) {
```

Comment:
Nit: I find this easier to process. And gets a bit easier if the `skip<>` is a `fail<>` :).
```suggestion
		if (!success && !skipBuildFailureOnRegression) {
```

Reply:
done